app/internal/page/about_payment.go

```diff
@@ -26,7 +26,7 @@ func AboutPayment(logger Logger, tmpl template.Template, sessionStore sessions.S
 
 		if r.Method == http.MethodPost {
 			createPaymentBody := pay.CreatePaymentBody{
```

Comment:
Have you seen https://docs.payments.service.gov.uk/optional_features/welsh_language/?

Reply:
Perfect - I've updated this now to define welsh for the language setting and return URL.